app/models/issue_token.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class IssueToken < ApplicationRecord
```

Comment:
I might question, why do we need to store a single global token in the database rather than just putting it in the credentials file?

Reply:
> I might question

As a matter of fact, you did indeed question

Reply:
The only way to _get_ the token is through the web callback flow.

Some other options, I guess:

* Figure out how to edit the encrypted credentials from within the controller action (seems dangerous)
* `File.write` the value to some _other_ configuration file.
* Just output the token value in the rendered view and make Adam copy-and-paste that to the production credentials file.

Reply:
Oh I see, I've worked with these tokens in development (specifically, my own from the round-three production database) but haven't looked into the actual generation process. I guess assumed you could somehow generate them directly on github.

Expanding on the third option, we might be able to do the generation process entirely in a development environment?

I guess the only point of doing all that would be to get rid of chunks of code that you've already written though.

Reply:
Yeah, and it makes it into a thing that is... within @sherson's capabilities, but he'll probably need help getting the app set up and a walk through on encrypted credential editing, etc. As is, he just has to click a "yes" button.